portfolio/src/main/webapp/index.html

```diff
@@ -110,11 +110,11 @@ <h2>Got something to say?</h2>
           <form id="comments-form" action="/data" method="POST">
             <div class="form-element">
               <label>Nickname</label>
-              <input class="comment-io" name="nickname" type="text" placeholder="Sam S" pattern="\w+">
+              <input class="comment-io" name="nickname" type="text" placeholder="Sam S">
             </div>
             <div class="form-element">
               <label>So, what'd you think?</label>
-              <input class="comment-io" name="comment" type="text" placeholder="I thought it was pretty cool, but..." pattern="\w+" required>
+              <input class="comment-io" name="comment" type="text" placeholder="I thought it was pretty cool, but..." pattern=".+" required>
```

Comment:
I believe the presence of `required` makes this specific `pattern` unneeded. You can just drop `pattern` here.

Reply:
Pattern dropped fe59a8d